kubernetes/provider.go

```diff
@@ -194,6 +194,10 @@ func Provider() *schema.Provider {
 			// storage
 			"kubernetes_storage_class":    dataSourceKubernetesStorageClass(),
 			"kubernetes_storage_class_v1": dataSourceKubernetesStorageClass(),
+
+			// admission control
+			"kubernetes_mutating_webhook_configuration":    dataSourceKubernetesMutatingWebhookConfiguration(),
```

Comment:
Let's remove this, since we didn't have an unversioned one already.

Reply:
Fixed 👍🏾